source/about/licensing-and-subscription.rst

```diff
@@ -1,125 +1,31 @@
-Licensing and Subscription
---------------------------
+=====================
+Subscription Overview
+=====================
 
-Mattermost offers a `secure self-service Customer Portal <https://customers.mattermost.com>`__ where you can easily purchase and manage your Mattermost Enterprise subscriptions.
+Mattermost offers features through `Starter, Professional, or Enterprise subscription plans <www.mattermost.com/pricing>`__. Your subscription to a plan determines what features you have access to. When choosing a subscription, you can choose the deployment type - self-hosted or cloud, and the plan - Starter, Professional, or Enterprise. 
 
-When you purchase a subscription, a license is generated. This license gives you access to Mattermost's Enterprise features. Subscriptions purchased via the Customer Portal are stored and listed in the portal, with license details including their start and end dates, providing you with an at-a-glance overview of your account. You also have full access to your billing history, making it easier to manage purchases and renewal dates.
+Mattermost self-hosted requires a license to be applied to access features in the Professional or Enterprise plans. Mattermost offers a `secure self-service Customer Portal <https://customers.mattermost.com>`__ where you can easily purchase and manage your Mattermost self-hosted subscriptions. When you purchase a subscription, a license is generated. Please see :ref:`self-hosted-subscriptions` for more details. 
 
-You can access your Customer Portal account to view information about your:
+Mattermost Cloud is a software-as-as-service, and you can simply sign up and start using Mattermost in a trial mode and upgrade to the edition you desire within the product. Please see :ref:`cloud-subscriptions` for more details.
 
-- Subscription purchases
-- Licenses
-- Customer Portal account password
-- Organization information
-- Payment methods
-- Renewals
-- Active users (available in a future release)
+For more general information and frequently asked questions on licensing and subscriptions, please see below. 
```

Comment:
```suggestion
For more general information and frequently asked questions on licensing and subscriptions, please see the following sections. 
```

Reply:
Applied in new file created when I changed the file name.